src/container_stadtnavi.json

```diff
@@ -2973,28 +3106,28 @@
       },
       "lookup_table": [],
       "default_value": "",
-      "created_date": "2024-12-20 09:53:51",
-      "updated_date": "2024-12-20 09:53:51",
-      "created_date_pretty": "Dec 20, 2024 10:53:51",
-      "updated_date_pretty": "Dec 20, 2024 10:53:51"
+      "created_date": "2024-12-20 13:30:45",
+      "updated_date": "2024-12-20 13:30:45",
+      "created_date_pretty": "Dec 20, 2024 14:30:45",
+      "updated_date_pretty": "Dec 20, 2024 14:30:45"
     },
     {
-      "idvariable": 953,
+      "idvariable": 1040,
       "type": "CustomJsFunction",
       "name": "Page URL with locations removed",
-      "description": "",
+      "description": "Remove user-specific data from the tracking URL",
       "parameters": {
-        "jsFunction": "function() {\n  const path = window.location.pathname;\n  const query = window.location.href.split(\"?\")[1] || \"\";\n  const parts = path.split(\"/\");\n  const removed = \"[removed]\"\n\n  // the search URLs contain locations which we strip out\n  if(path.indexOf(\"/reiseplan\") == 0){\n    parts[2] = removed;\n    parts[3] = removed;\n  }\n\n  // near you and favourites also has locations to be stripped\n  else if(parts[3] == \"indernaehe\" || parts[3] == \"gespeichertesuchen\"){\n    parts[1] = removed;\n    parts[2] = removed;\n  }\n\n  const modifiedPath = parts.join(\"/\");\n  return window.location.protocol + \"//\" + window.location.host + modifiedPath + \"?\" + query;\n\n};"
+        "jsFunction": "function() {\n  const path = window.location.pathname;\n  const query = window.location.href.split(\"?\")[1] || \"\";\n  const parts = path.split(\"/\");\n  const removed = \"[removed]\"\n\n  // the search URLs contain locations which we strip out\n  if(path.indexOf(\"/reiseplan\") == 0){\n    parts[2] = removed;\n    parts[3] = removed;\n  }\n\n  // near you and favourites also has locations to be stripped\n  else if(parts[3] == \"indernaehe\" || parts[3] == \"gespeichertesuchen\"){\n    parts[1] = removed;\n    parts[2] = removed;\n  }\n\n  query = removed; // removes e.g. \"setTime=true&time=1734876900\"\n\n  const modifiedPath = parts.join(\"/\");\n  return window.location.protocol + \"//\" + window.location.host + modifiedPath + \"?\" + query;\n\n};"
```

Comment:
After opening https://dev.stadtnavi.eu/, the following error pops up repeatedly. Lookin into this line, reassigning removed to the `const query` seems to be the culprit. Please check.

<img width="963" alt="grafik" src="https://github.com/user-attachments/assets/93cdd93f-caf2-4db3-9d5b-85006d494d33" />